src/frontend/src/lib/types/token.ts

```diff
@@ -40,6 +40,7 @@ export type OptionTokenStandard = TokenStandard | undefined | null;
 export type TokenToPin = Pick<Token, 'id'> & { network: Pick<Token['network'], 'id'> };
 
 interface TokenFinancialData {
+	formattedBalance?: number;
```

Comment:
If we want to hold the formatted balance, shouldn't it be a `string` then or is there a use case to convert it to number?

Reply:
Yes, it is even more consistent with the name